SignalServiceKit/src/Messages/DeviceSyncing/OWSSyncGroupsMessage.m

```diff
@@ -58,7 +59,10 @@ - (NSData *)buildPlainTextAttachmentDataWithTransaction:(YapDatabaseReadTransact
         enumerateCollectionObjectsWithTransaction:transaction
                                        usingBlock:^(id obj, BOOL *stop) {
                                            if (![obj isKindOfClass:[TSGroupThread class]]) {
-                                               DDLogVerbose(@"Ignoring non group thread in thread collection: %@", obj);
+                                               if (![obj isKindOfClass:[TSContactThread class]]) {
```

Comment:
We know there are contact threads in the same collection as group threads - this was cluttering up the logs.